CondFormats/DQMObjects/src/HDQMSummary.cc

```diff
@@ -54,9 +54,9 @@ const HDQMSummary::Range HDQMSummary::getRange(const uint32_t& DetId) const {
   RegistryIterator p = std::lower_bound(indexes_.begin(), indexes_.end(), DetId, HDQMSummary::StrictWeakOrdering());
   if (p == indexes_.end() || p->detid != DetId) {
     return HDQMSummary::Range(v_sum_.end(), v_sum_.end());
-    std::cout << "not in range " << std::endl;
-  } else
-    return HDQMSummary::Range(v_sum_.begin() + p->ibegin, v_sum_.begin() + p->ibegin + userDBContent_.size());
+    //std::cout << "not in range " << std::endl;
```

Comment:
Also here,  can we keep the printout (moving it above the return and making it a `LogWarning`?

Reply:
sure we can if this is required. see my comment https://github.com/cms-sw/cmssw/pull/43030#discussion_r1361606548